spec/support/http_handling_shared.rb

```diff
@@ -148,6 +148,18 @@
         expect(sockets_used.uniq.length).to eq(1)
       end
 
+      context "on a mixed state" do
+        it "re-opens the connection" do
+          first_socket = client.get("#{server.endpoint}/socket/1").body.to_s
```

Comment:
Huh? This is the socket ID you made the request with,
https://github.com/httprb/http.rb/blob/master/spec/support/dummy_server/servlet.rb#L61.
The response body is the socket ID.

On Mon, Mar 30, 2015 at 8:42 AM, Alexey Zapparov <notifications@github.com>
wrote:

> In spec/support/http_handling_shared.rb
> <https://github.com/httprb/http.rb/pull/194#discussion_r27401849>:
>
> > @@ -148,6 +148,18 @@
> >          expect(sockets_used.uniq.length).to eq(1)
> >        end
> >
> > +      context "on a mixed state" do
> > +        it "re-opens the connection" do
> > +          first_socket = client.get("#{server.endpoint}/socket/1").body.to_s
>
> With all respect but how first_socket_id = <...>.body.to_s makes it more
> clear? You are saving first socket's response body into a variable named
> first_socket_id. That makes as much sense as naming it first_socket. IMO
> even naming variables a and b will be less confusing in this case.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/httprb/http.rb/pull/194/files#r27401849>.
>